docs/xxflightrecorder.md

```diff
@@ -61,6 +61,49 @@ jcmd 1234 JFR.start filename=/path/ABCD.jfr duration=4s
 
 ```
 
+All JFR-related `jcmd` options might change in future releases.
+
+:fontawesome-solid-triangle-exclamation:{: .warn aria-hidden="true"} **Limitations:**
+
+- JFR produces only the following events:
+
+```
+- ClassLoadingStatistics
+- CPUInformation
+- CPULoad
+- ExecutionSample
+- GCHeapConfig
+- InitialEnvironmentVariable
+- InitialSystemProperty
+- JVMInformation
+- MonitorEnter
+- MonitorWait
+- NativeLibrary
+- OSInformation
+- PhysicalMemory
+- SystemGC
+- SystemProcess
+- ThreadContextSwitchRate
+- ThreadCPULoad
+- ThreadDump
+- ThreadEnd
+- ThreadPark
+- ThreadSleep
+- ThreadStart
+- ThreadStatistics
+- VirtualizationInformation
+- YoungGenerationConfig
+
+```
+- Support for the following capabilities is not available:
```

Comment:
capabilities ~is~ are not

Reply:
Focus is on the "Support..." therefore "Support is not available"